src/app/content/components/Page/PageComponent.tsx

```diff
@@ -67,12 +67,16 @@ export default class PageComponent extends Component<PagePropTypes> {
     this.highlightManager = highlightManager(this.container.current, () => this.props.highlights, this.props.services, this.props.intl);
     this.scrollToTopOrHashManager = scrollToTopOrHashManager(this.container.current);
 
+    // Sometimes data is already populated on mount, eg when navigating to a new tab
     if (this.props.searchHighlights.selectedResult) {
       this.searchHighlightManager.update(null, this.props.searchHighlights, {
         forceRedraw: true,
         onSelect: this.onSearchHighlightSelect,
       });
     }
+    if (this.props.scrollToTopOrHash.hash) {
+      this.scrollToTopOrHashManager({...this.props.scrollToTopOrHash, hash: ''}, this.props.scrollToTopOrHash);
```

Comment:
🤔🤔🤔🤔🤔🤔

i think i again want this to be null on the first argument

the structure of this hash isn't even really in scope for this code, it may even be better to just call it unconditionally and let the internal logic decide if it needs to do anything